docs/docs/config.md

```diff
@@ -0,0 +1,64 @@
+---
+name: "Configuration Reference"
+route: "/config"
+---
+
+# Configuration Reference
+
+Tanka's behavior can be customized per Environment using a file called `spec.json`
+
+## File format
+
+```json
```

Comment:
```suggestion
```js
```

Just a drive-by suggestion from a random person on the internet 😅 Because comments are not valid JSON, Github really doesn't seem to like rendering this block with a JSON syntax specified ...

![Screenshot 2020-04-20 at 12 29 10](https://user-images.githubusercontent.com/2285130/79746890-a33bcd00-8302-11ea-96fc-4c5cda927236.png)

Reply:
True, it looks ugly on GitHub, but renders better on our website when having it as json instead of js. The website is powered by the vscode highlight engine, which deals fine with this:

https://deploy-preview-251--tanka.netlify.app/config#file-format

<img width="1167" alt="Screen Shot 2020-04-20 at 14 50 21" src="https://user-images.githubusercontent.com/26971117/79753471-4518e500-8316-11ea-87a8-3bab059b9184.png">